apkid/rules/apk/packers.yara

```diff
@@ -534,3 +530,22 @@ rule apkpacker : packer
     condition:
         is_apk and all of them
 }
+
+rule chornclickers : packer
+{
+
+  meta:
+    description = "ChornClickers"
+    explanation = "ChornClickers ('Ch'-china '-orn'-porn '-clickers')"
```

Comment:
Sorry I wasn't more clear about this -- I just meant to add something like:
```// This has no name so we made one up from 'Ch'-china '-orn'-porn '-clickers'```

Reply:
Done